README.md

```diff
@@ -43,23 +45,22 @@ Activate the virtual environment:
 
 Install all dependencies into the virtual environment:
 
-```bash
-pip install -r requirements.txt
-```
+- `pip install -r ../requirements.txt`
 
 #### Run migrate and load fixtures
 
+
 Migrate the database to the latest schema:
 
-- `python ./manage.py migrate`
+- `python manage.py migrate`
 
-Mock data can be loaded into your dev site with the following command
+By default, Django sets the site domain to `example.com`, but the mock data needs the domain to be `localhost:8000`. Run the following command to update the site domain automatically
 
-- `python ./manage.py loaddata network-api/networkapi/fixtures/test_data.json`
+- `python manage.py update_site_domain`
```

Comment:
Not directly related to this PR but I got errors after running `python manage.py update_site_domain`  (fresh install)

```bash
(venv) mavisou:~/Development/network-test/network-api[master*]$ python manage.py update_site_domain
Traceback (most recent call last):
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: django_site

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/Users/mavisou/Development/network-test/network-api/networkapi/management/commands/update_site_domain.py", line 9, in handle
    site = Site.objects.get(id=1)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 379, in get
    num = len(clone)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 238, in __len__
    self._fetch_all()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 1087, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 835, in execute_sql
    cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: django_site
```

Reply:
Not sure if the fix is to update the command in README or fix the script? @gvn @gideonthomas

Reply:
hmm you shouldn't be seeing this if you ran `python manage.py migrate` first. That's weird

Reply:
kk. I ran `python manage.py migrate` again and it fixed the problem.